src/style.css

```
@@ -1 +1,75 @@
 /* add styles here */
+body {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+  color: #3D4E61;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 300;
+  line-height: normal;
```

Default values, so you can just remove it
```suggestion
  font-weight: 300;
```